content/blog/2021/04/2021-04-15-jenkins-operator-sub-project.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Jenkins Operator becomes an official sub-project!"
+tags:
+- kubernetes
+- operator
+- contributing
+authors:
+- antoniaklja
+- sylwiabrant
+sig: cloud-native
+opengraph:
+  image: https://virtuslab.com/wp-content/uploads/2019/04/jenkins_y6uwlf.jpg
```

Comment:
not sure if we want an external image here? is there precedent? It wouldn't be good if this breaks in future.

Reply:
I believe Oleg meant we could use the image, but not necessarily linking directly to it.

Reply:
fixed 👍